help/eess/index.md

```diff
@@ -0,0 +1,19 @@
+# Electrical Engineering and Systems Science archive
+
+The Electrical Engineering and Systems Science (eess) archive formed 18 September 2017 and you can read about the [announcement of archive here](/new/eess_announce).
+
+You can browse articles and view categories at the [main economics archive page](/archive/eess).
+
+The moderators for economis are [listed here](/moderators#eess).
```

Comment:
```suggestion
The [moderators for economics are listed here](/moderators#eess).
```

Reply:
same as previous suggestions... tiny typo, and the longer link has more meaning for screen readers